pandas/tests/indexes/datetimes/test_ops.py

```diff
@@ -231,15 +231,15 @@ def test_order_without_freq(self, index_dates, expected_dates, tz_naive_fixture)
         index = DatetimeIndex(index_dates, tz=tz, name="idx")
         expected = DatetimeIndex(expected_dates, tz=tz, name="idx")
 
-        ordered = index.sort_values()
+        ordered = index.sort_values(na_position="first")
```

Comment:
NaTs used to be sorted to the beginning by default (because their numerical representation is -big_number). Now they get sorted to the end by default, and the user needs to supply the argument to get previous behavior. Wonder if this should be noted in the docs somewhere.